internal/environment/environment.go

```diff
@@ -54,15 +54,31 @@ func NewPaths(base string) Paths {
 // BasePath returns krew base directory.
 func (p Paths) BasePath() string { return p.base }
 
+// IndexBase returns the krew index directory. This directory contains the default
+// index and custom ones.
+func (p Paths) IndexBase() string {
+	return filepath.Join(p.base, "index")
+}
+
 // IndexPath returns the base directory where plugin index repository is cloned.
 //
 // e.g. {BasePath}/index/
-func (p Paths) IndexPath() string { return filepath.Join(p.base, "index") }
+func (p Paths) IndexPath(name string) string {
+	if _, ok := os.LookupEnv(constants.EnableMultiIndexSwitch); ok {
+		if name == "" {
```

Comment:
if you use `"default"` in the call sites instead of `""` you can simply eliminate this `name=""` check, as well. :)